client/packages/requisitions/src/ResponseRequisition/DetailView/columns.ts

```diff
@@ -56,36 +60,178 @@ export const useResponseColumns = () => {
         accessor: ({ rowData }) => rowData.itemStats.availableStockOnHand,
       },
     ],
-    {
+  ];
+
+  if (!programName) {
+    columnDefinitions.push({
       key: 'customerStockOnHand',
       label: 'label.customer-soh',
       description: 'description.customer-soh',
       width: 100,
       align: ColumnAlign.Right,
+      Cell: PackQuantityCell,
       getSortValue: rowData =>
-        rowData.linkedRequisitionLine?.itemStats?.availableStockOnHand ?? '',
+        linkedRequisition
```

Comment:
Think we should stop the users from inputting calculated values if we have the data to calculate it

Reply:
If I'm reading this right - a response requisition with a linked requisition will have item stats that report 0?

![image](https://github.com/user-attachments/assets/caf6a2b7-b4a7-43ac-a07e-8c0cc998238a)

Hmm nah! I tried to debug this but can't follow the backend through itemStats, just goes into async-graphql futures seemingly forever hahaha

Curious why the MOS calculation is different result though!

Reply:
In a request requisition the AMC and available_SOH from item_stats is saved to the requisition line, so you are correct. We can use that instead of the itemStats.

Yeah, you're right. A response requisition with linked requisition will have item stats that report 0, but do we want the user to be able to input item stats (like AMC and SOH) for internal stores if we are saving their item stats on the requisition line?